HW5.py

```diff
@@ -0,0 +1,20 @@
+import csv, json
+parser = argparse.ArgumentParser()
+parser.add_argument('-csv')
+parser.add_argument('-json')
+ns = parser.parse_args(sys.argv[1:])
+
+csvFilePath = "user_details.csv"
+jsonFilePath = "user_details.json"
+
+
+data = {}
+with open(csvFilePath) as csvFile:
+    csvReader = csv.DictReader(csvFile)
+    fieldnames = [0]
+    for csvRow in csvReader:
+         user_id = csvRow['user_id']
+         data[user_id] = csvRow
+         data[user_id]["password"] = "NULL"
+         with open('jsonFilePath','w') as jsonFile:
+            jsonFile.write(json.dumps(data, indent = 4))
```

Comment:
Your JSON file rewrites 10 times because it locates in **for** loop. 
For example, on iteration #10 the code will clear all written 9 items, then add the next row to dict and write 10 items.

So this line should be used outside of the loop.

Reply:
There are some minor differences between **dumps** and **dump**:
> **json.dump** serializes obj as a JSON formatted stream to fp (a .write()-supporting file-like object.
> **json.dumps** serializes obj to a JSON formatted str

So **dump** will be a little bit more appropriate in this case:

`json.dump(data, jsonFile, indent=4)`

Reply:
It would be better to add the "main" function